Controller/ConnectController.php

```diff
@@ -296,7 +297,8 @@ protected function getErrorForRequest(Request $request)
      */
     protected function getResourceOwnerByName($name)
     {
-        $ownerMap = $this->container->get('hwi_oauth.resource_ownermap.'.$this->container->getParameter('hwi_oauth.firewall_name'));
+        // TODO: fix firewall_names[0]
+        $ownerMap = $this->container->get('hwi_oauth.resource_ownermap.'.$this->container->getParameter('hwi_oauth.firewall_names')[0]);
```

Comment:
But this will be quite slow... maybe this should be refactored to be set on runtime in extension?